pkg/commands/resolver.go

```diff
@@ -139,7 +139,10 @@ func makePublisher(po *options.PublishOptions) (publish.Interface, error) {
 			// TODO(jonjohnsonjr): I'm assuming that nobody will
 			// use local with other publishers, but that might
 			// not be true.
-			return publish.NewDaemon(namer, po.Tags), nil
+			return publish.NewDaemon(namer,
```

Comment:
Does me comment above here still hold up? Would skaffold ever want to do local + remote?

Reply:
By local+remote, you mean both side-loading to a Docker daemon _and_ publishing to a registry? If so, it still holds AFAIK. Skaffold has a boolean `push` config to either side-load or publish: https://skaffold.dev/docs/references/yaml/#build-local-push

Reply:
> you mean both side-loading to a Docker daemon and publishing to a registry?

Yeah, it seems like an unlikely use case, but I'm sure someone might want to do that, eventually.

>  so, it still holds AFAIK. Skaffold has a boolean push config to either side-load or publish

perfect, thanks 👍